PrayerBook/Orison.swift

```diff
@@ -8,9 +8,36 @@
 
 import Foundation
 
-struct Orison {
+struct Orison: Codable {
```

Comment:
bring protocol subscription to extension

Reply:
ok